### PR TITLE
Simple feature implemented

### LINKED ISSUE #19: Create a simple feature to have a skeleton to work
The idea is to create a super simple feature to trigger the movement of the project.

I consider having a starting point is a good thing to iterate and make things happen.

### PR DESCRIPTION
**Pull request by soygabimoreno**
### :tophat: What is the goal?

**To have a starting point.**

The idea is to create a super simple feature to trigger the movement of the project.
I consider having a starting point is a good thing to iterate and make things happen.

This should be migrated to Jetpack Compose.
In fact, I consider it is a good idea to let that process in the scope of a single PR.

BTW, I didn't add any tests because I feel we are going to get rid of many of this and I prefer to wait for reviews.

This close #19

### CODE REVIEW

**Review by Monzonprog — APPROVED** [review]

**Review by cbedoy — APPROVED** [review]
All good

**Review by Adrc95 — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -43,19 +48,45 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    viewBinding {
+        enabled true
+    }
 }
 
 dependencies {
     implementation 'androidx.core:core-ktx:1.7.0'
```
Comment (jmanday) [inline app/build.gradle:57]: What about include the navigation dependency?

Comment (cbedoy) [reply app/build.gradle:57]: I think it is a simple feature integration and one single fragment, hence integrate navigation component will increase complexity. Will be worth if has more than one fragments

Comment (jmanday) [reply app/build.gradle:57]: Ok, we can add it when we need use it

Comment (soygabimoreno) [reply app/build.gradle:57]: I am not very fond of the navigation library.
Anyway, we can start using it, if we would need it.

**Pull request merged**